Updated README to correctly state default '--pool-size'

pawelad opened the pull request:
I might be wrong, but just in case I'm not, I _think_ the actual default value of `--pool-size` is 4, and not the 8 stated in the README:

https://github.com/wal-e/wal-e/blob/8d1eefba556d9e9f160ada2dc0b39a062b4e70b6/wal_e/cmd.py#L246-L248

deverant commented:
That section of the README actually refers to the wal-push command. The readme is still in error since the actual pool-size for that command is now 32 by default.

pawelad commented:
> That section of the README actually refers to the wal-push command. The readme is still in error since the actual pool-size for that command is now 32 by default.

Ah, you're right - my bad.

I rebased the brach, fixed the value and the version it was changed. I also added a note about using `--pool-size` in `backup-fetch` and `backup-pull` - not sure if that's the right place, so feel free to let me know and I'll delete it.

pawelad commented:
Bump
